tox.ini

```diff
@@ -20,6 +20,7 @@ whitelist_externals =
     g++
     ldd
 extras =
```

Comment:
With  the current settings `dace` is always installed, which means that a vanilla `gt4py` installation is never tested.
I would suggest to add a new factor to the matrix to run either vanilla or dace-flavored gt4py (and also add a similar change in  `.github/workflows/gt4py-tox.yml` to run both branches in parallel)

```
envlist =
    py{38,39}-{internal,dace}-{cpu,cuda,cuda90,cuda91,cuda92,cuda100,cuda101}

extras =
    testing
    dace: dace
    cuda: cuda
    cuda90: cuda90
    cuda91: cuda91
    cuda92: cuda92
    cuda100: cuda100
    cuda101: cuda101

[testenv:py{38,39}-{internal,dace}-cpu]
...

[testenv:py{38,39}-{internal,dace}-{cuda,cuda90,cuda91,cuda92,cuda100,cuda101}]
...

```

Reply:
I think Enrique proposes internal backends only and internal **plus** dace

Reply:
> I think Enrique proposes internal backends only and internal **plus** dace
Yes, that was my proposal.

Reply:
I read it that way too, but then I thought of all the CI time we already spend 😆